### PR TITLE
Split border attribute

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: src/main/java/de/retest/web/WebElementPeer.java [hunk 1]
```diff
@@ -122,14 +122,14 @@ protected MutableAttributes retrieveStateAttributes() {
 		final MutableAttributes state = new MutableAttributes();
 		for ( final String attribute : AttributesProvider.getInstance().getAttributes() ) {
 			final String attributeValue = webData.get( attribute );
-			if ( attributeValue != null && !isDefault( attributeValue ) ) {
+			if ( attributeValue != null && !isDefault( attribute, attributeValue ) ) {
 				state.put( attribute, attributeValue );
 			}
 		}
 		return state;
```
Comment (retest-sonar-cloud) [inline src/main/java/de/retest/web/WebElementPeer.java]: Code Smell Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=32&issues=AWSLWuHMsk71k7lwmQ83&open=AWSLWuHMsk71k7lwmQ83)

[ISSUE_KEY]: AWSLWuHMsk71k7lwmQ83

File: src/main/java/de/retest/web/WebElementPeer.java [hunk 2]
```diff
@@ -126,6 +126,9 @@ private boolean isDefault( final String attributeValue ) {
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
+		if ( AttributesProvider.getInstance().getSizes().contains( attribute ) && attributeValue.equals( "0px" ) ) {
```
Comment (retest-sonar-cloud) [inline src/main/java/de/retest/web/WebElementPeer.java]: Code Smell Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=32&issues=AWSLWuHMsk71k7lwmQ83&open=AWSLWuHMsk71k7lwmQ83)

[ISSUE_KEY]: AWSLWuHMsk71k7lwmQ83

File: src/main/java/de/retest/web/WebElementPeer.java [hunk 3]
```diff
@@ -126,10 +126,7 @@ private boolean isDefault( final String attribute, final String attributeValue )
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
-		if ( AttributesProvider.getInstance().getSizes().contains( attribute ) && attributeValue.equals( "0px" ) ) {
-			return true;
-		}
-		return false;
+		return AttributesProvider.getInstance().getSizes().contains( attribute ) && attributeValue.equals( "0px" );
```
Comment (beatngu13) [inline src/main/java/de/retest/web/WebElementPeer.java]: I wouldn't do this. As you already said, this is not an improvement but a false positive from SonarCloud. Would you deleting the commit? I can show you how to fix this in SonarCloud.

**Comment by beatngu13** [comment]
There's still some work to do, I'll take care.

**Pull request closed without merging**